web/src/components/navigation.js

```diff
@@ -38,27 +39,50 @@ const NavigationLink = styled.a`
 
 
 class Navigation extends Component {
+ 
   render() {
     return (
+      <Router>
       <NavigationList>
         <NavigationListItem>
-          <NavigationLink href="/">Dashboards</NavigationLink>
+            <NavigationLink href="/">
+                <Link to="/">Dashboards</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/coops">Co-Ops</NavigationLink>
+            <NavigationLink href="/coops">
+                <Link to="/coops/">Co-Ops</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/members">Members</NavigationLink>
+            <NavigationLink href="/members">
+                <Link to="/members/">Members</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/transactions">Transactions</NavigationLink>
+            <NavigationLink href="/transactions">
+                <Link to="/transactions/">Transactions</Link>
+            </NavigationLink>
           </NavigationListItem>
         <NavigationListItem>
-          <NavigationLink href="/loans">Loans</NavigationLink>
+            <NavigationLink href="/loans">
+                <Link to="/loans/">Loans</Link>
+            </NavigationLink>
           </NavigationListItem>
       </NavigationList>
+
+      <Route path="/" exact component={this.Index} />
+      <Route path="/coops/" component={this.Coops} />
+      <Route path="/members/" component={this.Members} />
```

Comment:
Once this is moved to App.js, if we change this to `component={Content}` (and get rid of the `this.` for this and transactions) or  rename the `Content` component to `Members`, then we should be able to render the components that should already be imported at the top of the App.js file.